### PR TITLE
Use data stream to verify references

### PR DESCRIPTION
**Pull request by jan-cerny**
We will use SCAP source data stream as an input for verify-references
test instead of the XCCDF 1.1 file. We will have to do small updates
in verify_references.py. This change will help us to remove outdated
XCCDF 1.1 from the project.

### CODE REVIEW

File: build-scripts/verify_references.py [hunk 1]
```diff
@@ -209,7 +207,9 @@ def main():
             # "https://" values (since the "name" attribute will be empty for
             # these two cases)
             if check_content_ref_href_attr.startswith("http://") or \
-               check_content_ref_href_attr.startswith("https://"):
+               check_content_ref_href_attr.startswith("https://") or \
+               check_content_ref_href_attr.startswith("security-data-oval-com.redhat.rhsa-") or \
```
Comment (vojtapolasek) [inline build-scripts/verify_references.py:211]: Please update also the comment just above this conditional.

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Good job! Please fix issues reported by Codeclimate. And a small nitpick, see comment.

**Comment by jan-cerny** [comment]
I have updated the comment, fixed the code climate problems and rebased this pull request on the top of the latest upstream master branch.

**Comment by jan-cerny** [comment]
I have avoid very long lines by renaming varaible

**Review by vojtapolasek — APPROVED** [review]
Looks good, thank you.

**Pull request merged**